File: backend/task_manage_app/src/main/java/edu/dilini/ecom/dto/Task.java
package edu.dilini.ecom.dto;

import com.fasterxml.jackson.annotation.JsonFormat;
import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import jakarta.persistence.Temporal;
import jakarta.persistence.TemporalType;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Task {
    private Long id;
    private String title;
    private String description;
    private String status;
    @JsonFormat(pattern = "MM-dd-yyyy:HH:mm")
    private LocalDateTime createdAt;
}

File: forntend/task-manage-app/src/app/model/Task.ts
export class Task{
    
    title:string;
    description:string;
    status:string;
    createdAt:Date

    constructor(title:string,description:string,status:string,createdAt:Date){
        
        this.title=title;
        this.description=description;
        this.status=status;
        this.createdAt=createdAt;
    }

}